src/cryptoadvance/specter/util/psbt.py

```diff
@@ -0,0 +1,364 @@
+"""
+The goal of this module is to slowly migrate from json-like representation of PSBT received from Bitcoin RPC
+to a normal PSBT class that does not require RPC calls and can do more things.
+to_dict and from_dict methods are maintained for backward-compatibility
+"""
+from embit.psbt import PSBT, InputScope, OutputScope, DerivationPath
+from embit.transaction import Transaction, TransactionOutput
+from embit.liquid.networks import get_network
+from embit import bip32
+from math import ceil
+import time
+
+
+class SpecterTx:
+    def __init__(self, parent, tx):
```

Comment:
If we migrate here from json to classes, can't we also use static types in here? If a complete newbie Dev looks at this class, he has no idea what "parent" and "tx" should be even though in the `_to_dict` method, he learns that it needs to have a `txid-method`.
So maybe we can use static type checking for all the constructors of these "value-types" ?
https://medium.com/@ageitgey/learn-how-to-use-static-type-checking-in-python-3-6-in-10-minutes-12c86d72677b

Reply:
Done. I hope it will be easier to understand now.